Resources/public/js/profiling/GABackend.js

```diff
@@ -59,13 +59,17 @@ Ext.define('Modera.backend.googleanalytics.profiling.GABackend', {
         }
 
         if (target instanceof MF.activation.activities.AbstractActivity) {
-            ga('send', {
-                hitType: 'timing',
-                timingCategory: 'MJR',
-                timingVar: this.trackingPlugin.createToken(),
-                timingLabel: 'View activation',
-                timingValue: ms
-            });
+            if (typeof ga !== 'undefined') {
```

Comment:
This won't cut it. If you look below in the method, you will see a second invocation of ga() method. Create a delegate method in this class that would check in a single place if ga() is globally defined and invoke it, otherwise just, say, ignore.